Doppler.Sap/Enums/CountryCodeEnum.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.Sap.Enums
+{
+    public enum CountryCodeEnum
+    {
+        AR,
+        US
+    }
```

Comment:
Is it fine using enums? or you should use a constant? 

For example, this code would be safer if you use a constant:

![image](https://user-images.githubusercontent.com/1157864/98970426-b1e01080-24ee-11eb-8082-187a7cc8ab55.png)

(or you can also convert the enum value to a string, to avoid the magic strings)